template/src/report.js

```diff
@@ -47,9 +49,12 @@ export class Report {
 
     document.getElementById('report').innerHTML = html;
 
-    $('.bar').on('click', (event) => {
-      this.handleBarClick(event);
-    });
+    const bars = Array.from(document.querySelectorAll('.bar'));
+    for (const bar of bars) {
+      bar.addEventListener('click', (event) => {
```

Comment:
Seems like deregistering the event handler wasn't done previously as well!?

Reply:
No. I don't think it is really necessary, since these event listeners are needed as long as the DOM bar elements are available. Once these elements are destroyed, the GC of the browser will clean up the attached listeners (although I don't know how good is this working in IE11). For an example report, should be ok to let it like this, in my opinion.